### ISSUE TITLE
How to open/use the extension?

### ISSUE THREAD

**Issue by CodeCubeNeo** [YouTube-Enhancer/extension#606]
How to open the extension? Please add this information to readme.md

**Comment by VampireChicken12** [comment]
@CodeCubeNeo are you talking about how to open the extension settings?

**Comment by CodeCubeNeo** [comment]
no, that is logical (you click the icon in the browser toolbar). For example i want to loop the video. Nothing appears on youtube.com

**Comment by VampireChicken12** [comment]
You have to enable the settings for stuff to appear. The extension doesn't come with an opinionated default configuration

**Comment by VampireChicken12** [comment]
> You have to enable the settings for stuff to appear. The extension doesn't come with an opinionated default configuration

@CodeCubeNeo
